debian/rules

```diff
@@ -23,4 +23,4 @@ override_dh_auto_test:
 	true
 
 override_dh_virtualenv:
-	dh_virtualenv $EXTRA_PYPI --python=/usr/bin/python2.7 --preinstall no-manylinux1 --preinstall pip-custom-platform --pip-tool pip-custom-platform
+	dh_virtualenv $EXTRA_PYPI --python=/usr/bin/python2.7 --preinstall pip-custom-platform --pip-tool pip-custom-platform
```

Comment:
you need `no-manylinux1`, manylinux wheels are not compatible with dh-virtualenv

this essentially defeats the purpose of this entire review :P